packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -365,7 +382,7 @@ test.serial('writeGranules() throws an error if collection is not provided', asy
   );
 });
 
-test.serial('writeGranules() saves granule records to Dynamo and Postgres if Postgres write is enabled', async (t) => {
+test.serial('writeGranules() saves granule records to Dynamo/PostgreSQL/Elasticsearch if PostgreSQL write is enabled', async (t) => {
```

Comment:
```suggestion
test.serial('writeGranules() saves granule records to DynamoDB/PostgreSQL/Elasticsearch if PostgreSQL write is enabled', async (t) => {
```
really minor but it keeps it consistent